homeassistant/components/wallbox/helpers.py

```diff
@@ -0,0 +1,59 @@
+"""Helpers for Wallbox integration."""
+
+from datetime import datetime as datetime_sys
+
+from dateutil import tz
+
+import homeassistant.util.dt as dt_util
+
+
+def map_schedule_item_to_ha(item: dict) -> dict:
+    """Map local schedule item to be more usable with HA."""
+    converted = {}
+    converted["start"] = _convert_to_local_time(item["start"])
+    converted["stop"] = _convert_to_local_time(item["stop"])
+    converted["id"] = item["id"]
+    converted["enable"] = item["enable"]
+    converted["days"] = item["days"]
+    if "max_current" in item:
```

Comment:
this can be a 1-liner I think, same for the other if's

Reply:
Most probably yes. I'm not very experienced with python. Mind sharing an example how to do this as one-liner?

Reply:
I think that you don't need to do anything if you just use the original dict right? but otherwise:

converted["max_energy" = item.get("max_energy") (this deals with missing values)